docs/source/developers/cpp/building.rst

```diff
@@ -39,7 +39,7 @@ out-of-source. If you are not familiar with this terminology:
 
 Building requires:
 
-* A C++17-enabled compiler. On Linux, gcc 7.1 and higher should be
+* A C++17-enabled compiler. On Linux, gcc 9 and higher should be
```

Comment:
What do you think about making this change in a separate issue and PR so it's more visible?

Reply:
[Issue](https://github.com/apache/arrow/issues/47025) and [PR](https://github.com/apache/arrow/pull/47026).